tools/plugins/com.liferay.ide.project.ui/META-INF/MANIFEST.MF

```diff
@@ -7,7 +7,7 @@ Bundle-Activator: com.liferay.ide.project.ui.ProjectUI
 Bundle-Vendor: %bundle.vendor
 Require-Bundle: org.eclipse.ui,
  org.eclipse.jst.common.project.facet.ui;bundle-version="[1.4.1,1.5.0)";visibility:=reexport,
```

Comment:
Can we change all of the version ranges to go until the next breaking change?  That will make us more future proof

Reply:
Can you make this change across all of our manifests?  Thanks.

Reply:
ok , I will remove all version ranges.